pl-faded-parsons-question.mustache

```diff
@@ -23,7 +23,7 @@
         </div>
         {{/scrambled}}
         {{#pre_text}}
-        <pre class="prettyprint" language="{{{language}}}"> {{{text}}} </pre>
+        <pre class="prettyprint" {{#language}}language="{{{language}}}"{{/language}}> {{{text}}} </pre>
```

Comment:
have we checked this works? Inner "{{{language}}}" might need be to "{{..}}" or something like it